.github/workflows/upload_latest_files.yml

```diff
@@ -1,47 +1,46 @@
-name: Upload latest install related files to CDN
```

Comment:
Why do we comment out this file? This action takes no effect now.

Reply:
Because the dashboard (UI) doesn‘t sync all the changes. Before all components are available, this action will be commented out.

The changes brought by nirvana are **disruptive** for the UI. And also, It’s equally important to keep the `API` functionally synchronized before modifying the UI. We didn't do these two things well.

When the upstream changes are too large, the work of API is also very difficult (**No documentation and lack of comments**). Apart from this, our current `API` is extremely fragmented and doesn't follow a certain fixed rule, such as `restful`. And we didn't maintain the unity of the interface, such as the abstraction of public fields. **These problems will be more obvious in the face of major changes.**

These problems will have a chain reaction. If these problems persist, the front-end will also be forced to constantly modify and adapt. Also, when a bug appears, the front end needs to feedback upstream and try to solve the problem. All of these will block the development of the front-end.

I think after this PR, our team has recognized this problem. We will make profound improvements to this.